src/ansible_navigator/configuration_subsystem/definitions.py

```diff
@@ -134,16 +138,22 @@ class SubCommand(SimpleNamespace):
     epilog: Union[None, str] = None
 
 
-class ApplicationConfiguration(SimpleNamespace):
+@dataclass
+class ApplicationConfiguration:
+    # pylint: disable=too-many-instance-attributes
     """The main object for storing an application config"""
 
-    application_name: str = ""
-    entries: List[SettingsEntry] = []
+    application_version: str
```

Comment:
Prefer adding fields in a separate PR instead.

Reply:
No choice really, it was already in use by the NavigatorConfiguration:

```
NavigatorConfiguration = ApplicationConfiguration(
    application_name=APP_NAME,
    application_version=VERSION,
```

Reply:
Sorry, I missed explaining that...

Reply:
Gotcha. Resolving then.